Tasks/AzureSpringCloudV0/package-lock.json

```diff
@@ -468,7 +468,7 @@
         "xmldom": "^0.1.27"
       }
     },
-    "node_modules/azure-pipelines-tasks-webdeployment-common-v4/node_modules/@types/mocha": {
+    "node_modules/azure-pipelines-tasks-webdeployment-common/node_modules/@types/mocha": {
```

Comment:
Just double confirm: it is by design to drop the "v4" instead of make it "v5", right?

Reply:
There might be some conflict, let me double check.